services/ui-src/src/components/fields/DateField.tsx

```diff
@@ -13,6 +13,7 @@ export const DateField = ({
   hint,
   sxOverride,
   nested,
+  autosave, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
could just remove this line since it's not being used. Then there's no need for the comment